File: OOP C# Course/DefineClassesExercise/15.Drawingtool/Models/Rectangle.cs
﻿namespace Drawingtool.Models
{
    using System;

    public class Rectangle
    {
        public int width;
        public int height;

        public Rectangle(int width, int height)
        {
            this.width = width;
            this.height = height;
        }

        public void Draw()
        {
            for (int i = 0; i < this.height; i++)
            {
                if (i == 0 || i == this.height - 1)
                {
                    Console.WriteLine("|" + new string('-', this.width) + "|");
                }
                else
                {
                    Console.WriteLine("|" + new string(' ', this.width) + "|");
                }
            }
        }
    }
}

File: JavaScript Core/JS Advanced/Classes And Class Memebers/Lab/Rectangle.js
class Rectangle{
    constructor(width,height,color){
       this.width = width
       this.height= height
       this.color = color
    }

    calcArea(){

        return Number(this.width) * Number(this.height)
    }
}

let rect = new Rectangle('20', '30', '40')

console.log(rect.calcArea());